module/zfs/dmu_send.c

```diff
@@ -62,21 +63,26 @@
 int zfs_send_corrupt_data = B_FALSE;
 int zfs_send_queue_length = 16 * 1024 * 1024;
 int zfs_recv_queue_length = 16 * 1024 * 1024;
+/* Set this tunable to FALSE to disable setting of DRR_FLAG_FREERECORDS */
+int zfs_send_set_freerecords_bit = B_TRUE;
```

Comment:
You wouldn't want it set to FALSE.  See https://illumos.org/issues/6536 for this history of this flag.